KarhooUISDK/Screens/BookingScreen/QuoteListMVP/KarhooQuoteListViewController.swift

```diff
@@ -207,17 +207,18 @@ final class KarhooQuoteListViewController: UIViewController, QuoteListView {
     func categoriesChanged(categories: [QuoteCategory], quoteListId: String?) {
         quoteCategoryBarView.categoriesChanged(categories: categories, quoteListId: quoteListId)
     }
+
+    func toggleSortingFilteringControls(show: Bool) {
```

Comment:
Bit pedantic but can we maybe call it something like setCategoryFilteringControlsVisibility, toggleCategoryFilteringControls or something like that?

Reply:
Sure, no problem :) renamed it to `toggleCategoryFilteringControls`